pkg/fleet/installer/repository/repository.go

```diff
@@ -366,16 +367,10 @@ func movePackageFromSource(ctx context.Context, packageName string, rootPath str
 	if err := os.Chmod(targetPath, 0755); err != nil {
 		return "", fmt.Errorf("could not set permissions on package: %w", err)
 	}
-	switch filepath.Base(rootPath) {
-	case "datadog-agent":
-		if err := service.ChownDDAgent(ctx, targetPath); err != nil {
+	if filepath.Base(rootPath) == "datadog-agent" {
+		if err := exec.Command("chown", "-R", "dd-agent:dd-agent", targetPath).Run(); err != nil {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏(non blocking) this should go in postinst